repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/guidance-content.ftl

```diff
@@ -85,17 +110,15 @@
                                 <#case "uk.nhs.hee.web.beans.CtaCardReference">
                                     <@hee.ctaCard ctaCard=block/>
                                     <#break>
+                                <#case "uk.nhs.hee.web.beans.ContentCards">
```

Comment:
This seem to render navigation cards at the end of the page rather than on the right hand side ?

Reply:
removed as not including in this release for rhs